src/applications/ivc-champva/shared/components/fileUploads/MissingFileList.jsx

```diff
@@ -91,7 +86,7 @@ export default function MissingFileList({
         const entryName = `${entry[nameKey].first} ${entry[nameKey]?.middle ||
           ''} ${entry[nameKey].last}${entry[nameKey]?.suffix || ''}`;
         return (
-          <div key={Object.keys(entry).join('') + idx}>
+          <div key={`${entryName}-${subset}`}>
```

Comment:
Fix for duplicate keys error being thrown by React - this will pair an applicant name with either 'required' or 'optional', which will give us unique keys (assumes first name, middle, lastname, and suffix together are sufficient for uniqueness)